data-raw/watershed/regional_approximation/Regional_Approximation.Rmd

```diff
@@ -16,37 +16,44 @@ library(lubridate)
 ```
 
 
-## Upper-mid Sacramento Region 
 
 ### Instream Spawning and Rearing Habitat
 
-**Details:**    
-A mean regional weighted usable area (WUA) was developed for watersheds without instream spawning and rearing habitat modeling. 
 
-For unmodeled tributaries of the Upper Sacramento River, the rearing WUA values were estimated using the mean WUA at each flow from Battle Creek, Butte Creek, Clear Creek, and Cow Creek. The spawning WUA values were estimated using the mean WUA at each flow from Battle Creek, Butte Creek, and Clear Creek.
 
-**Spawning Not Modeled:** 
+**Details:** 
 
-* Antelope Creek
-* Bear Creek
-* Big Chico Creek
-* Cow Creek
-* Elder Creek
-* Mill Creek
-* Paynes Creek
-* Stony Creek
-* Thomes Creek
+Habitat modeling is not available for several tributaries of the Upper Sacramento River. These tributaries share similar hydrologic and geomorphic conditions. In consultation with [Mark Gard](mailto:mark_gard@fws.gov) at the US Fish and Wildlife Service, regional approximations of instream spawning and rearing habitat were developed for tributaries without habitat modeling. The regional approximation for instream rearing habitat uses the mean WUA at each flow from Battle Creek, Butte Creek, Clear Creek, and Cow Creek. For instream spawning, the regional approximation uses the mean WUA at each flow from Battle Creek, Butte Creek, and Clear Creek.
 
-**Rearing Not Modeled:**    
+**The following table shows watershed areas and channel widths for all of the watersheds used to create regional approximations.**
 
-* Antelope Creek
-* Bear Creek
-* Big Chico Creek
-* Elder Creek
-* Mill Creek
-* Paynes Creek
-* Stony Creek
-* Thomes Creek
+```{r}
+tibble(
+  Watershed = c("Battle Creek", "Butte Creek", "Clear Creek", "Cottonwood Creek", 
+                "Cow Creek*"), 
+  `Area (sqkm)` = c(957, 2123, 645, 2444, 1107), 
+  `Width (m)` = c(22.5, 15.2, 18, 28.25, 8.5)
+) %>% 
+  knitr::kable(align = 'c')
+```
+
+*Cow Creek is not included in regional approximation for spawning. 
+
+**The following table shows watershed areas and channel widths for watersheds without spawning and/or rearing habitat modeling.** 
+
+
+```{r}
+tibble(
+  Watershed = c("Antelope Creek", "Bear Creek", "Big Chico Creek", "Cow Creek*", 
+                "Elder Creek", "Mill Creek", "Paynes Creek", "Stony Creek", 
+                "Thomes Creek"), 
+  `Area (sqkm)` = c(508, 649, 930, 1107, 358, 334, 243, 2021, 758), 
+  `Width (m)` = c(15.8, 4.3, 14.8, 8.5, 8.6, 14.3, 6.3, 19.6, 12.9)
+) %>% 
+  knitr::kable(align = 'c')
+```
+
+*Cow Creek does not have modeled spawning habitat. 
```

Comment:
Change to '*Cow Creek has modeled rearing habitat.'

Reply:
done